api/cmd/transformer/main.go

```diff
@@ -137,47 +147,44 @@ func main() {
 	s.Run()
 }
 
-func initFeastServingClients(feastOptions feast.Options, specs []*core.FeatureTableSpec, logger *zap.Logger) (feast.Clients, error) {
-	clients := feast.Clients{}
-
-	// Default Feast gRPC client. Used if user does not specify serving url
-	// on standard transformer config.
-	client, err := newFeastGrpcClient(feastOptions.DefaultServingURL)
-	if err != nil {
-		return nil, err
+func getServingType(userEnableDirectStorage *bool, currentServingType spec.ServingType) spec.ServingType {
+	servingType := currentServingType
```

Comment:
Can you simplify this part. It seems overly complicated where simple `if else` statement will do
```
if userEnableDirectStorage == nil {
  return currentServingType
}

if userEnableDirectStorage {
  return spec.ServingType_DIRECT_STORAGE
}

return spec.ServingType_FEAST_GRPC
```

Reply:
just realised it, will change